### PR TITLE
Remove color from evaluations ui center

### PR DESCRIPTION
**Pull request by johannkm**
image

Context: https://elementl-workspace.slack.com/archives/C04UD72HHQX/p1689007961856609

### CODE REVIEW

**Comment by johannkm** [comment]
Current dependencies on/for this PR:
* master
  * **PR #15200** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/15200" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/15200?utm_source=stack-comment).

**Comment by braunjj** [comment]
Looks good to me

File: js_modules/dagit/packages/ui/src/components/Colors.tsx [hunk 1]
```diff
@@ -42,4 +42,5 @@ export const Colors = {
   Olive500: 'rgba(156, 172, 151, 1)',
   Olive200: 'rgba(215, 222, 213, 1)',
   Olive50: 'rgba(236, 237, 236, 1)',
+  Black: 'rgba(0, 0, 0, 1)',
```
Comment (hellendag) [inline js_modules/dagit/packages/ui/src/components/Colors.tsx:45]: We should not use black anywhere. `Colors.Dark` is fine for our needs.

File: js_modules/dagit/packages/core/src/assets/AutoMaterializePolicyPage/Conditions.tsx [hunk 2]
```diff
@@ -11,25 +11,14 @@ import {AutoMateralizeWithConditionFragment} from './types/GetEvaluationsQuery.t
 
 export type ConditionType = AutoMateralizeWithConditionFragment['__typename'];
 
+const activeColor = Colors.Black;
```
Comment (hellendag) [inline js_modules/dagit/packages/core/src/assets/AutoMaterializePolicyPage/Conditions.tsx:14]: Use `Colors.Dark` in place of `activeColor`, delete this.

**Review by hellendag — CHANGES_REQUESTED** [review]

**Comment by johannkm** [comment]
Dark works if we make the non met text gray

Screenshot 2023-07-10 at 3 56 54 PM
Screenshot 2023-07-10 at 3 57 36 PM

**Review by hellendag — APPROVED** [review]

**Pull request merged**